third-party/snyk/snapshot.json

```diff
@@ -27,7 +27,8 @@
       {"id":"npm:backbone:20110701","severity":"medium","semver":{"vulnerable":["<0.5.0"]}}
     ],
     "bootstrap":[
-      {"id":"npm:bootstrap:20180529","severity":"medium","semver":{"vulnerable":[">=4.0.0 <4.1.2"]}},
```

Comment:
is it bad that [this vulnerability page](https://snyk.io/vuln/npm:bootstrap:20180529) has the same description and links as the [new vulnerability page](https://snyk.io/vuln/SNYK-JS-BOOTSTRAP-73560), but a different CVE number?

Seems weird, but either way the message is the same?